src/main/java/dev/openfeature/sdk/Structure.java

```diff
@@ -107,41 +113,6 @@ default Object convertValue(Value value) {
         throw new ValueNotConvertableError();
     }
 
-    /**
```

Comment:
This removal is very technically a breaking change, but it was never intended to be public and it's very awkward to use. It has more to do with `EvaluationContexts` not `Structures` anyway and I doubt it's widely used.